test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -417,10 +417,10 @@ test
   });
 
 test
-  .requestHooks(loginHintAppLinkMock)('expect login_hint in AppLink when engFastpassMultipleAccounts is on', async t => {
+  .requestHooks(loginHintAppLinkMock)('expect login_hint in AppLink in all cases', async t => {
```

Comment:
```suggestion
  .requestHooks(loginHintAppLinkMock)('expect login_hint in AppLink', async t => {
```

Reply:
I only recommend removing the `in all cases` part to simplify the test name because the usage of the flag is being completely removed, this now becomes the only case to consider.

Reply:
Sounds good. Done here and elsewhere.